Sources/DefaultPlugins/DebooggerButtonPlugin.swift

```diff
@@ -6,21 +6,25 @@ import Foundation
 
 final class DebooggerButtonPlugin: SwitchPlugin {
 
+    private enum Constants {
+        static let debooggerSuffix: String = "deboogger_"
+    }
+
     private var shouldShowDebooggerButton: Bool {
         get {
-            if let shouldShow = UserDefaults.standard.object(forKey: "deboogger_\(#function)") as? Bool {
+            if let shouldShow = UserDefaults.standard.object(forKey: Constants.debooggerSuffix + "\(#function)") as? Bool {
                 return shouldShow
             }
 
             var shouldShow = false
             #if targetEnvironment(simulator)
                 shouldShow = true
             #endif
-            UserDefaults.standard.set(shouldShow, forKey: "deboogger_\(#function)")
+            UserDefaults.standard.set(shouldShow, forKey: Constants.debooggerSuffix + "\(#function)")
             return shouldShow
         }
         set {
-            UserDefaults.standard.set(newValue, forKey: "deboogger_\(#function)")
+            UserDefaults.standard.set(newValue, forKey: Constants.debooggerSuffix + "\(#function)")
```

Comment:
still looks like code duplication by my opinion

Reply:
🤷‍♂️

Reply:
Moving **1 line of code**, which repeats twice, to separate function doesn't makes sense for me. Why is this so important to you?

Reply:
@ntunin ☝️